app/src/main/res/values/strings.xml

```diff
@@ -74,4 +73,9 @@
     <string name="title_add">Add</string>
     <string name="title_create">Create</string>
     <string name="action_x">x</string>
+    <string name="title_space_time">Space time</string>
+    <string name="title_end_date">End date</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_end_date` appears to be unused
 Unused resources make applications larger and slow down builds.